src/main/java/exceptions/advice/ApiError.java

```diff
@@ -0,0 +1,29 @@
+package exceptions.advice;
+
+import org.springframework.http.HttpStatus;
+
+public class ApiError {
+    private HttpStatus status;
+    private String message;
+
+    public ApiError(HttpStatus status, String message) {
+        this.status = status;
+        this.message = message;
```

Comment:
Add one additional string for the system message. {status_code : 403, status_message : Forbidden, message : you are not allowed to see this page }